hdrass/CCW+Rotator_integration_testing/Rotator_interlock_test.ipynb

```diff
@@ -0,0 +1,1248 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CCWstate = mtm.evt_cameraCableWrapFollowing.get()\n",
+    "print(\"CCWstate:\",CCWstate.enabled)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtm.cmd_enableCameraCableWrapFollowing.set_start(timeout=10.)\n",
+    "await CCWstate = mtm.evt_cameraCableWrapFollowing.next(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtm.cmd_disableCameraCableWrapFollowing.set_start(timeout=10.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Move to 0 deg Starting Position"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Test Start\")\n",
+    "\n",
+    "now = datetime.now()\n",
+    "script.evt_logMessage.set_put(level=logging.INFO+1,\n",
+    "                              message=f\"START - {test_message} - Move to 0 deg Starting Position - {now} UTC\")\n",
+    "\n",
+    "print(\"Move to 0.0 deg starting position\")\n",
+    "\n",
+    "#await rot.cmd_configureVelocity.set_start(vlimit=1.0)\n",
+    "await rot.cmd_move.set_start(position=0.0,timeout=30.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "alt = 45. * u.deg\n",
+    "az = 0. * u.deg\n",
+    "rot_tel = Angle(0, unit= u.deg) \n",
+    "\n",
+    "target_name=\"Rotator test\"\n",
+    "time_data = await mtptg.tel_timeAndDate.next(flush=True, timeout=2)\n",
+    "curr_time_mtptg = Time(time_data.mjd, format=\"mjd\", scale=\"tai\")\n",
+    "time_err = curr_time_mtptg - Time.now()\n",
+    "print(f\"Time error={time_err.sec:0.2f} sec\")\n",
+    "\n",
+    "# Compute RA/Dec for commanded az/el\n",
+    "cmd_elaz = AltAz(alt=alt, az=az, \n",
+    "                 obstime=curr_time_mtptg.tai, \n",
+    "                 location=location)\n",
+    "cmd_radec = cmd_elaz.transform_to(ICRS)\n",
+    "lst = Angle(time_data.lst, unit=u.hour)\n",
+    "\n",
+    "para_ang=parallactic_angle(location, lst, cmd_radec).to(u.deg)\n",
+    "print(f\"Alt: {alt}\\nAz: {az}\\nRotTel: {rot_tel}\\nParaAng: {para_ang}\")\n",
+    "\n",
+    "\n",
+    "# Calculating the other parameters     \n",
+    "rot_pa = para_ang-180*u.deg+rot_tel\n",
+    "print(rot_pa)\n",
+    "\n",
+    "await rot.cmd_stop.start(timeout=10.)\n",
+    "await asyncio.sleep(5.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"start tracking\")\n",
+    "await rot.cmd_trackStart.start(timeout=30.)\n",
+    "    \n",
+    "ack = await mtptg.cmd_raDecTarget.set_start(\n",
```

Comment:
Another useless `ack =`. flake8 would point out things like this. I it possible to run it on Jupyter notebooks? It would make your code more robust. Extra unused variables are safe (but confusing for readers). But flake8 also catches more important problems, such as undefined variables.

Reply:
From Michael: flake8 should be available since the LSST DM stack is installed on nublado.